ddtrace/monkey.py

```diff
@@ -147,6 +149,10 @@ def patch_module(module, raise_errors=True):
 
     Returns if the module got properly patched.
     """
+    if not formats.asbool(os.environ.get("DD_TRACE_%s_ENABLED" % module, True)):
+        log.info("Not patching %s as it's disabled by environment variable.", module)
```

Comment:
should we put in the env var?

Reply:
e.g. `disabled by environment variable DD_TRACE_%s_ENABLED`?

Reply:
Yeah, good idea!